core/src/main/java/org/commonjava/indy/data/DefaultStoreValidator.java

```diff
@@ -220,7 +222,7 @@ private ArtifactStoreValidateData compareRemoteHostToAllowedHostnames(Optional<U
 
     private ArtifactStoreValidateData availableSslRemoteRepository(CountDownLatch httpRequestsLatch,
                                                                    Optional<URL> remoteUrl,RemoteRepository remoteRepository) throws InterruptedException, ExecutionException, URISyntaxException {
-        HashMap<String, String> errors = new HashMap<>();
+//        HashMap<String, String> errors = new HashMap<>();
         // Execute HTTP GET & HEAD requests in separate thread pool from executor service
         Future<Integer> httpGetStatus = executeGetHttp(new HttpGet(remoteUrl.get().toURI()), httpRequestsLatch);
         Future<Integer> httpHeadStatus = executeHeadHttp(new HttpHead(remoteUrl.get().toURI()), httpRequestsLatch);
```

Comment:
FWIW, we need to be using the Indy HTTP subsystem to make these requests, too. It will use the remote repository definition, and can pull the server cert from that definition - if there is one - and use it during validation. A plain-vanilla HttpClient instance won't know about those, and could fail for valid (though rare) configurations.